engine/src/main/resources/org/camunda/bpm/engine/db/create/activiti.db2.create.engine.sql

```diff
@@ -112,6 +112,7 @@ create table ACT_RU_JOB (
     LOCK_OWNER_ varchar(255),
     EXCLUSIVE_ smallint check(EXCLUSIVE_ in (1,0)),
     EXECUTION_ID_ varchar(64),
+    ROOT_PROCESS_INSTANCE_ID_ varchar(64),
```

Comment:
I'm not sure if i agree on this statement because then it would break the convention for `ACT_RU_JOB` table which follows a `whole-name` convention.

Similarly, `PROCESS_INSTANCE_ID_` follows a whole-name convention for the job table but for all other tables, the convention is `PROC_INST_ID_`.

Since there is no unified convention for all tables, i'd vote to follow `ACT_RU_JOB`'s own convention.

Reply:
After discussing the point, we agreed to keep Yana's convention recommendation